catroid/src/main/java/org/catrobat/catroid/utils/HtmlRegexExtractor.java

```diff
@@ -27,19 +27,43 @@
 
 import org.catrobat.catroid.R;
 
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
 public class HtmlRegexExtractor {
+
 	private Context context;
+
 	public HtmlRegexExtractor(Context context) {
 		this.context = context;
 	}
-	public void searchKeyword(String search, String text) {
-		int index = text.indexOf(search);
+
+	public void searchKeyword(String keyword, String text) {
+		if (getKeywordIndex(keyword, text)) {
+			showSuccess();
+		} else {
+			showError();
+		}
+	}
+
+	public boolean getKeywordIndex(String keyword, String text) {
+		int index = text.indexOf(keyword);
 		if (index >= 0) {
-			ToastUtil.showSuccess(context,
-					R.string.formula_editor_function_regex_html_extractor_found);
+			return true;
 		} else {
-			ToastUtil.showError(context,
-					R.string.formula_editor_function_regex_html_extractor_not_found);
+			String keywordRegex = keyword.replace(" ", ".*");
```

Comment:
```suggestion
			String keywordRegex = keyword.replace(" ", ".*?");
```
Match a pattern that is as short as possible. This may avoid issues with multiple occurrences of the keywords.